variables.tf

```diff
@@ -755,19 +761,25 @@ variable "clusters" {
   description = "A list describing clusters workloads to create"
   type = list(
     object({
-      name               = string           # Name of Cluster
-      vpc_name           = string           # Name of VPC
-      subnet_names       = list(string)     # List of vpc subnets for cluster
-      workers_per_subnet = number           # Worker nodes per subnet.
-      machine_type       = string           # Worker node flavor
-      kube_type          = string           # iks or openshift
-      kube_version       = optional(string) # Can be a version from `ibmcloud ks versions` or `latest`
-      entitlement        = optional(string) # entitlement option for openshift
-      pod_subnet         = optional(string) # Portable subnet for pods
-      service_subnet     = optional(string) # Portable subnet for services
-      resource_group     = string           # Resource Group used for cluster
-      cos_name           = optional(string) # Name of COS instance Required only for OpenShift clusters
-      update_all_workers = optional(bool)   # If true force workers to update
+      name                            = string           # Name of Cluster
+      vpc_name                        = string           # Name of VPC
+      subnet_names                    = list(string)     # List of vpc subnets for cluster
+      workers_per_subnet              = number           # Worker nodes per subnet.
+      machine_type                    = string           # Worker node flavor
+      kube_type                       = string           # iks or openshift
+      kube_version                    = optional(string) # Can be a version from `ibmcloud ks versions` or `latest`
+      logdna_plan                     = optional(string) # Logging plan to provision
+      sysdig_plan                     = optional(string) # Monitoring plan to provision
+      disable_public_endpoint         = optional(bool)   # Flag indicating that the public endpoint should be disabled
+      verify_worker_network_readiness = optional(bool)   # Flag to run a script will run kubectl commands to verify that all worker nodes can communicate successfully with the master. If the runtime does not have access to the kube cluster to run kubectl commands, this should be set to false.
+      enable_platform_logs            = optional(bool)   # Receive platform logs in the provisioned IBM Cloud Logging instance.
+      enable_platform_metrics         = optional(bool)   # Receive platform metrics in the provisioned IBM Cloud Monitoring instance.
+      entitlement                     = optional(string) # entitlement option for openshift
```

Comment:
Suggest to keep the same variables as in the original here to simplify.

Reply:
pod_subnet and service_subnet are still defined here - but they are not used in the code from a quick glance.